pilot/pkg/xds/xds_test.go

```diff
@@ -300,26 +301,49 @@ func TestSidecarListeners(t *testing.T) {
 
 func TestMeshNetworking(t *testing.T) {
 	s := NewFakeDiscoveryServer(t, FakeOptions{
-		ConfigString: `
-apiVersion: networking.istio.io/v1alpha3
-kind: ServiceEntry
+		ObjectString: `
```

Comment:
nit: now that I see its a lit usage its a bit confusing if something is an "object" or a "config". Do you think we should automagically determine where it gets added to? Or rename it to RegistryString or KubernetesObjectString or something?